src/views/catalog/utils/quantity.ts

```diff
@@ -1,5 +1,8 @@
 import byteSize, { ByteSizeResult } from 'byte-size';
 
+const fixUnitSuffix = (unit: string): string =>
+  unit.charAt(unit.length - 1) === 'B' ? unit : unit + 'B';
```

Comment:
please try unit[length -1] i think we can skip chartAt

Reply:
We don't need this `fixUnitSuffix`, IMO `toIECUnit` implemented [here](https://github.com/kubevirt-ui/kubevirt-plugin/blob/main/src/utils/utils/units.ts#L14) already does the thing 😏

Reply:
Also a suggestion is to used xbytes package which is already into our code base. You can sample from the charts :)

Reply:
@hstastna Thanks for the tip on `toIECUnit`. I didn't know about it. I replaced `fixUnitSuffix` with it for this PR.

@metalice `xbytes` looks very clean. I'll open a separate PR to clean up this file where possible.

Reply:
cool, 😎  so if already cleaning, I think that toIECUnit can be replaced with xbytes so that we will be united within all our code base. looking forward to this PR. Thank you!